R/generate_template.R

```diff
@@ -313,3 +345,53 @@ create_new_application <- function(name, location, sampleapp = FALSE, resetbutto
 
     return()
 }
+
+create_default_theme_file <- function(theme_file) {
+    lines <- c("### primary_color",
+               "# To set primary status color, that affect the color of header, valueBox, infoBox and box",
+               "# Valid possible value are colors names or hex-decimal value (i.e,: \"blue\", \"#086A87\")",
+               "# Blank/empty value will use default value",
+               "primary_color: \n\n",
+               
+               
+               "# Sidebar variables: change the default sidebar width, colors:",
+               "### sidebar_width",
+               "# width is numeric value in pixels, must be greater than 0",
```

Comment:
Width is to be specified as a numeric value in pixels. Must be greater than 0 and include only the number. Blank/empty value will use default value

Reply:
done